docs/genesis.json

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
I would advise against putting the `genesis.json` file here and, instead, putting it into the [Morpheus repo](https://github.com/desmos-labs/morpheus) to keep everything more organized.

Reply:
updated the url